content/docs/04.5-devx/03-cluster-groups/35-ingress-cluster-group.md

```diff
@@ -0,0 +1,93 @@
+---
+title: "Set Up Ingress"
+metaTitle: "Set Up Ingress for Cluster Groups"
+metaDescription: "Learn how to set up a Kubernetes Ingress for a Palette Cluster Group"
```

Comment:
Suggestion: "Learn how to configure Ingress for a Palette Cluster Group"

Users are not setting up **a Kubernetes Ingress** (singular) in this guide. We should use "Ingress" because it is plural.

Reply:
Done